community_server/src/Controller/StateBalancesController.php

```diff
@@ -84,200 +85,68 @@ public function overview()
         //}
         //
         //
+        $stateBalancesTable = TableRegistry::getTableLocator()->get('StateBalances');
         $stateUserTransactionsTable = TableRegistry::getTableLocator()->get('StateUserTransactions');
+        $transactionsTable  = TableRegistry::getTableLocator()->get('Transactions');
+        
+        
+        $stateBalancesTable->updateBalances($user['id']);
 
-        $creationsTable = TableRegistry::getTableLocator()->get('TransactionCreations');
-        $creationTransactions = $creationsTable
-                ->find('all')
-                ->where(['state_user_id' => $user['id']])
-                ->contain(['Transactions']);
-
-        $transferTable = TableRegistry::getTableLocator()->get('TransactionSendCoins');
-        $transferTransactions = $transferTable
-                ->find('all')
-                ->where(['OR' => ['state_user_id' => $user['id'], 'receiver_user_id' => $user['id']]])
-                ->contain(['Transactions']);
-
-        $involvedUserIds = [];
-
-        foreach ($transferTransactions as $sendCoins) {
-          //var_dump($sendCoins);
-            if ($sendCoins->state_user_id != $user['id']) {
-                array_push($involvedUserIds, intval($sendCoins->state_user_id));
-            } elseif ($sendCoins->receiver_user_id != $user['id']) {
-                array_push($involvedUserIds, intval($sendCoins->receiver_user_id));
-            }
-        }
+        $gdtSum = 0;        
+        $gdtEntries = $this->JsonRequestClient->sendRequestGDT(['email' => $user['email']], 'GdtEntries' . DS . 'sumPerEmailApi');
```

Comment:
Whats this `'GdtEntries' . DS . 'sumPerEmailApi'`? is this call documented? `[...]GdtEntries/sumPerEMailApi`

Reply:
no, it isn't documented, because it is part of the GDT API. 
You cannot call it from frontend because of access destrictions. 
It allows only calls from known domains.